### ISSUE TITLE
Choose where to insert references instead of always being at the end

### ISSUE THREAD

**Issue by ps-george** [Zestylogic/FMark#190]
Like TOC. e.g.

```
%% REFS
```

If no %% REFS in doc, can default to inserting references at the bottom of the document.

**Comment by ymherklotz** [comment]
What if you don't want any references printed though? I would probably prefer if you always had to specify `%% REFS` if you want references, as one might have references in the markdown without wanting to print them sometimes?

Let me know what you think though.

**Comment by ps-george** [comment]
You're right - I agree with you completely. I'll edit the issue description.
